critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -101,6 +101,11 @@ def test_draft_review(self):
         response = self.client.get("/review/%s" % review["id"])
         self.assert404(response, "Drafts shouldn't be publicly visible.")
 
+        data = {review["entity_type"]: review["entity_id"]}
+        response = self.client.get('/review/write/', query_string=data)
+        redirect_url = urlparse(response.location)
+        self.assertEquals(redirect_url.path, url_for('review.edit', id=review['id']))
```

Comment:
I still don't see a test that redirects /review/write?artist=1234 -> /review/write/artist/1234 in the case that no review exists yet.

Reply:
oh wait, that's the one I asked for in https://github.com/metabrainz/critiquebrainz/pull/298#issuecomment-691968954, sorry!

Reply:
Yup, and that test is also added at https://github.com/metabrainz/critiquebrainz/pull/298/files#diff-68c7d6f53d1f78aada84956c9906f4e7R143.